src/medication-request/medication-request.resource.tsx

```diff
@@ -87,11 +90,15 @@ function buildPrescriptionsTableRow(
     patientName: encounter?.subject?.display,
     drugs: [
       ...new Set(
-        medicationRequests.map((o) =>
-          o.medicationReference
-            ? o.medicationReference.display
-            : o.medicationCodeableConcept?.text
-        )
+        medicationRequests
+          .map((o) =>
+            o.medicationReference
+              ? o.medicationReference.display
+              : o.medicationCodeableConcept?.text
+          )
+          .sort((a, b) => {
+            return a.localeCompare(b);
+          })
```

Comment:
provide consistent ordering in display... I noticed cases where it was changing during refreshes, which looked strange.